### DISCUSSION TITLE
Modern Industrialization Battery Alloy

### DISCUSSION THREAD

**Discussion by HacoKavanah** [TechReborn/TechReborn#2949] [Show and tell]
Would be neat to be able to make battery alloy plates from that mod in a tech reborn compressor!

**Comment by Ayutac** [comment]
how are they done in MI? Link to the recipe in their repo would be perfect :)

**Reply by Ayutac** [reply]
oooor I just do complete compatibility with MI

**Reply by HacoKavanah** [reply]
Sounds hot

**Comment by Ayutac** [comment]
Implemented in MI, PR is here: https://github.com/AztechMC/Modern-Industrialization/pull/288
